src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -758,9 +758,18 @@ static int rbd_nbd(int argc, const char *argv[])
 
   argv_to_vec(argc, argv, args);
   env_to_vec(args);
-  auto cct = global_init(NULL, args, CEPH_ENTITY_TYPE_CLIENT,
-			 CODE_ENVIRONMENT_DAEMON,
-			 CINIT_FLAG_UNPRIVILEGED_DAEMON_DEFAULTS);
+
+  code_environment_t code_env = CODE_ENVIRONMENT_UTILITY;
+  int env_flag = 0;
+  for (auto arg : args) {
+    if (strcmp(arg, "map") == 0) {
```

Comment:
@trociny , global_init will handle common args, and remove the argument start with"-" from args, so that the code behind global_init could just use args.begin():
if (strcmp(*args.begin(), "map") == 0) 
If we just move global_init to do_map, how do we check args.begin()? maybe many changes...